admin_manual/maintenance/upgrade.rst

```diff
@@ -2,93 +2,47 @@
 How to Upgrade Your ownCloud Server
 ===================================
 
-There are three ways to upgrade your ownCloud server:
+We recommend that you keep your ownCloud server upgraded regularly installing all point releases and major releases, without skipping any of them, as doing so increases the risk of errors. 
+
+- Major releases are 8.0, 8.1, 8.2, 9.0, and 10.0. 
+- Point releases are intermediate releases for each major release, e.g., 8.0.9, 8.1.3, and 10.0.1. 
```

Comment:
I know these are only example numbers, but since 10.0.1 only existed in real life for a short time, perhaps use 10.0.2 as the example.